scripts/thanks.js

```diff
@@ -0,0 +1,33 @@
+if (!remoteUrl) {
+  var remoteUrl = "https://api.carpoolvote.com/live";
+}
+
+const handleBackendResponse = () => {
+  var data = tinyQuery.getAll();
```

Comment:
Yes, very aware of the dependency. Part of the reason for my comments on jekyll above and why the code has been tested. Still, the code should either work completely or fail completely, so that an inconsistent state is never shown to the user. In the failure state, the site behaviour is as it is without this PR.
Anyway, resolved by your jekyll tip